build.gradle

```diff
@@ -1,33 +1,56 @@
 buildscript {
   repositories { jcenter() }
-  dependencies { classpath 'com.netflix.nebula:gradle-rxjava-project-plugin:4.0.0' }
+  dependencies { 
+    classpath 'com.netflix.nebula:gradle-rxjava-project-plugin:4.0.0' 
+    classpath 'ru.vyarus:gradle-animalsniffer-plugin:1.1.0'
+  }
 }
 
 group = 'io.reactivex.rxjava2'
 
 description = 'RxJava: Reactive Extensions for the JVM – a library for composing asynchronous and event-based programs using observable sequences for the Java VM.'
 
 apply plugin: 'java'
-apply plugin: 'pmd'
+// apply plugin: 'pmd'
```

Comment:
hm, that's sad

On Tue, Jul 19, 2016 at 12:33 AM, David Karnok <notifications@github.com>
wrote:

> In build.gradle
> <https://github.com/ReactiveX/RxJava/pull/4204#discussion_r71234422>:
>
> >  }
> >
> >  group = 'io.reactivex.rxjava2'
> >
> >  description = 'RxJava: Reactive Extensions for the JVM – a library for composing asynchronous and event-based programs using observable sequences for the Java VM.'
> >
> >  apply plugin: 'java'
> > -apply plugin: 'pmd'
> > +// apply plugin: 'pmd'
>
> Our build task gets killed most of the time if it is enabled.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/ReactiveX/RxJava/pull/4204/files/a9752a3d697c31ad1ac46bb41891777038e0b0d8#r71234422>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA7B3CFN7hn3Q5ju6HxbFuMF08iCxKEWks5qW_ElgaJpZM4JMU7W>
> .
>